Clients/Xamarin.Interactive.Client/ClientApp/WebApp.targets

```diff
@@ -15,8 +15,7 @@
   </PropertyGroup>
 
   <Target
-    Name="_Browserify"
-    DependsOnTargets="ResolveNodeCli">
+    Name="_Browserify">
     <Exec WorkingDirectory="$(MSBuildThisFileDirectory)" Command="&quot;$(Npm)&quot; install --save-dev" />
```

Comment:
s/Npm/Yarn now?

Reply:
No. I am not touching our old stuff to make it work with yarn.